.github/workflows/linters.yml

```diff
@@ -0,0 +1,76 @@
+name: Linters
+
+on: pull_request
```

Comment:
- [ ] Please notice that you are passing any tests. I can see that you added `linters.yml` file, but except them you should have `.stylelintrc.json `and `.hintrc` files to make the tests work. Lets make the tests work! Please follow the link and make sure to complete all the steps: https://github.com/microverseinc/linters-config/tree/master/html-css